extensions/amp-story/1.0/bookend/amp-story-bookend.js

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+import {AMP_CUSTOM_LINKER_TARGET} from '../../../../src/service/navigation';
```

Comment:
Services can be loaded asynchronously and should not be bundled in the extension code. It'd just add this service code into the amp-story extension, even though it's already present in the v0 file.
You could either have a getter in the navigation service, or just duplicate the constant